### PR TITLE
Hide preview of follow events in dashboard

### LINKED ISSUE #894: Hide follow events
Hide or not?

X followed Y

Class is `.follow`

### PR DESCRIPTION
**Pull request by AshikNesin**
Fixes #894 

**Before**
image

**After**

image

### CODE REVIEW

File: source/features/hide-follow-events-preview.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import select from 'select-dom';
+import {observeEl} from '../libs/utils';
+
+// Hide the preview of follow events in dashboard
+export default function () {
+	observeEl('#dashboard .news', () => {
+		for (const item of select.all('#dashboard .news .follow .body .border')) {
```
Comment (AshikNesin) [inline source/features/hide-follow-events-preview.js]: Not sure this is the proper way of targeting the DOM element :(

Any suggestions?

**Comment by lucasbento** [comment]
This PR addresses the same thing as #899.

**Comment by bfred-it** [comment]
Yeah, duplicate of #899

**Comment by AshikNesin** [comment]
Oops. I should have checked PR before doing it :)

**Comment by lucasbento** [comment]
@AshikNesin: thank you for the contribution though!

File: source/features/hide-follow-events-preview.js (hunk 1 above)
Comment (bfred-it) [reply source/features/hide-follow-events-preview.js]: Yep! It’s correct ☺️

Comment (AshikNesin) [reply source/features/hide-follow-events-preview.js]: I'll make the change and submit new PR tonight :)

**Pull request closed without merging**